src/tests/test_jsproxy.py

```diff
@@ -745,24 +728,21 @@ def test_jsmod_import_star2(selenium):
 
 @pytest.mark.skip_refcount_check
 @pytest.mark.skip_pyproxy_check
-def test_nested_import(selenium_standalone):
-    selenium = selenium_standalone
-    assert (
-        selenium.run_js(
-            """
-            self.a = { b : { c : { d : 2 } } };
-            return pyodide.runPython("from js.a.b import c; c.d");
-            """
-        )
-        == 2
-    )
-    selenium.run(
-        """
-        import sys
-        del sys.modules["js.a"]
-        del sys.modules["js.a.b"]
-        """
-    )
+@run_in_pyodide
+def test_nested_import(selenium):
+    import js
+    from pyodide.code import run_js
+
+    js.a = run_js("() => ({ b : { c : { d : 2 } } })")()  # type: ignore[attr-defined]
```

Comment:
The point of this one is to test importing from `js` so we should keep the assignment in JavaScript.
```suggestion
    from pyodide.code import run_js

    run_js("globalThis.a = { b : { c : { d : 2 } } }") 
```